tests/UpgradeRule/PHP/fixtures/update-visibility.testfixture

```diff
@@ -0,0 +1,26 @@
+{
+}
+------
+<?php
+
+use SilverStripe\ORM\DataObject;
+
+class Animal extends DataObject
+{
+    public static $db = [
+        'Name' => 'Varchar'
+    ];
+
+    public static $do_not_change_me = "foobar";
+}
+------
+<?php
+
+use SilverStripe\ORM\DataObject;
+
+class Animal extends DataObject
+{
+    private static $db = array('Name' => 'Varchar')
```

Comment:
The end results always appear to saved on a single line. For most fields, especially `$db`, this is goign to be very painful. Can we get it to retain it's previous formatting?